sys-cluster/pmix/pmix-5.0.3.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI=8
 inherit flag-o-matic libtool
 
 DESCRIPTION="The Process Management Interface (PMI) Exascale"
-HOMEPAGE="https://openpmix.github.io/"
+HOMEPAGE="https://openpmix.github.io https://github.com/openpmix/openpmix"
```

Comment:
This is unnecessary, since it's not really the "homepage" but anyways the information is in metadata.xml

I don't really think it's great to try to one by one add the contents of metadata.xml to every single ebuild as a side effect of submitting PRs to bump the version.

Reply:
Fine. I've updated the PR.